.travis.yml

```diff
@@ -87,6 +87,11 @@ before_script:
     fi
 script:
   - |
+    if [[ "$PHPCS" == "1" ]]; then
+      composer phpcs
+      bash php-lint.sh
```

Comment:
PHP linting `php-lint.sh` should preferably be done on all PHP versions. Now you're only checking if there are no parse errors on PHP 7.3, which doesn't mean there won't be any parse errors on lower PHP versions.